views/shared/_navigation.haml

```diff
@@ -0,0 +1,4 @@
+-if current_user
+  %form.inline{action: '/user_sessions/logout', method: 'POST'}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace